darts/models/forecasting/forecasting_model.py

```diff
@@ -1893,7 +1893,12 @@ def _get_model_description_string(self, include_default_params):
         changed_model_params = [
             (k, v)
             for k, v in self.model_params.items()
-            if include_default_params or v != default_model_params.get(k, None)
+            if include_default_params
+            or (
+                isinstance(v, (pd.DataFrame, np.ndarray))
+                and np.any(v != default_model_params.get(k, None))
+            )
+            or v != default_model_params.get(k, None)
```

Comment:
I think we can use np in general
```suggestion
            if include_default_params
            or np.any(v != default_model_params.get(k, None))
```